src/services/user-service.ts

```diff
@@ -0,0 +1,61 @@
+import { SignupModel } from "../models/signup-model";
+import { LoginModel } from "../models/login-model";
+import { HashManager } from "../utils/hash-manager";
+import { PrismaClient } from "@prisma/client";
+import { TokenManager } from "../utils/token-manager";
+import { ApiError } from "../error/api-error";
+import { createLoginResponse } from "../models/login-response-model";
+import { createSignupResponseModel } from "../models/signup-response-model";
+
+const hashManager = new HashManager();
+const prisma = new PrismaClient();
+const tokenManager = new TokenManager();
+
+class UserService {
```

Comment:
this class should be tested.
but I'm not sure if this is a later step in the training